core/src/main/resources/log4j.properties

```diff
@@ -8,5 +8,6 @@ log4j.logger.kafka=ERROR, stdout
 log4j.logger.org.apache.zookeeper=ERROR, stdout
 log4j.logger.org.apache.kafka=ERROR, stdout
 log4j.logger.org.I0Itec.zkclient=ERROR, stdout
+log4j.logger.org.apache.directory=ERROR, stdout
```

Comment:
`org.apache.directory` logs started being created as part of Kerberos testing. This suppresses those logs to just show ERROR message. INFO is super chatty.